code/modules/religion/burden.dm

```diff
@@ -0,0 +1,165 @@
+#define FIRST_TRUTH_REWARD 3
+#define SECOND_TRUTH_REWARD 6
+
+#define BAD_MUTATIONS_REQUIRED 3
+
+/datum/religion_sect/burden
+	name = "Punished God"
+	desc = "A sect that desires to feel the pain of their god."
+	alignment = ALIGNMENT_NEUT
+	convert_opener = "\"To feel the freedom, you must first understand captivity.\"<br>\
+	Incapacitate yourself in any way possible. Bad mutations, lost limbs, traumas, even addictions. You will learn the secrets of the universe \
```

Comment:
you didnt make traumas count for it, and even if it did i think you reach 5 stuff at max and not 6

Reply:
see wip checkbox